src/jdk.crypto.ec/share/classes/sun/security/ec/SunEC.java

```diff
@@ -193,182 +191,128 @@ public Void run() {
 
     void putEntries(boolean useFullImplementation) {
         HashMap<String, String> ATTRS = new HashMap<>(3);
-
-        if (FIPSConfigurator.enableFips()) {
-            ATTRS.put("ImplementedIn", "Software");
-            String ecKeyClasses = "java.security.interfaces.ECPublicKey" +
-                    "|java.security.interfaces.ECPrivateKey";
-            ATTRS.put("SupportedKeyClasses", ecKeyClasses);
-            ATTRS.put("KeySize", "256");
-
-            /*
-             * Key Factory engine
-             */
-            putService(new ProviderService(this, "KeyFactory",
-                "EC", "sun.security.ec.ECKeyFactory",
-                new String[] { "EllipticCurve" }, ATTRS));
-
-            /*
-             * Algorithm Parameter engine
-             */
-            // "AlgorithmParameters.EC SupportedCurves" prop used by unit test
-            boolean firstCurve = true;
-            StringBuilder names = new StringBuilder();
-            Pattern nameSplitPattern = Pattern.compile(CurveDB.SPLIT_PATTERN);
-
-            Collection<? extends NamedCurve> supportedCurves =
-                CurveDB.getSupportedCurves();
-            for (NamedCurve namedCurve : supportedCurves) {
-                if (!firstCurve) {
-                    names.append("|");
-                } else {
-                    firstCurve = false;
-                }
-
-                names.append("[");
-
-                String[] commonNames = nameSplitPattern.split(namedCurve.getName());
-                for (String commonName : commonNames) {
-                    names.append(commonName.trim());
-                    names.append(",");
-                }
-
-                names.append(namedCurve.getObjectId());
-                names.append("]");
+        ATTRS.put("ImplementedIn", "Software");
+        String ecKeyClasses = "java.security.interfaces.ECPublicKey" +
+                 "|java.security.interfaces.ECPrivateKey";
+        ATTRS.put("SupportedKeyClasses", ecKeyClasses);
+        ATTRS.put("KeySize", "256");
+
+        /*
+         * Key Factory engine
```

Comment:
Please revert unnecessary changes relative to 9494bb5ffb761591c7e04a8a4b5b3380bee05a91; here and on line 301.

Reply:
Those changes have not been reverted.